src/assets/images/landscape/comparison-of-service-mesh-strengths.svg

```diff
@@ -1 +1 @@
0-11.2,0 c-0.6,0-1.1,0-1.7,0c-0.1,0-0.3,0-0.3-0.1c0,0,0,0,0-0.1c0,0,0,0,0,0c0-0.3,0-0.6-0.1-0.9c-0.1-0.5-0.1-0.5,0.4-0.5 c2.2-0.3,4.4-0.6,6.5-0.8c0.4,0,0.7-0.1,1.1-0.1c2.1-0.3,4.3-0.5,6.4-0.8c0.7-0.1,1.4-0.2,2.1-0.3c0.1,0,0.3,0,0.4,0 c1-0.1,2.1-0.3,3.1-0.4c0.9-0.1,1.8-0.2,2.7-0.4c0.3,0,0.7-0.1,1-0.1c0.2,0,0.3,0,0.3,0.2c0,0.2,0,0.3,0.1,0.5 c0,0.2,0,0.3-0.2,0.3c-1.3,0.5-2.7,0.9-4,1.4c-1.1,0.4-2.2,0.8-3.4,1.2c0,0-0.1,0-0.1,0c-0.1,0-0.2,0-0.3,0.1 C1022.1,124.8,1021.4,125.1,1020.6,125.4L1020.6,125.4z M960,80.7C960,80.7,960,80.7,960,80.7C960,80.7,960,80.7,960,80.7 L960,80.7z" class="st8"/><path d="M1017.4,86c0.1-0.1,0.1-0.2,0.1-0.3l0,0c0.1-0.1,0.1-0.2,0.2-0.4l0,0c0.1-0.2,0.2-0.5,0.3-0.8l0,0v0 c0.2-0.3,0.3-0.7,0.4-1c0.1-0.2,0.2-0.4,0.3-0.7l0,0c0.5-1,0.9-2.1,1.3-3.1v0c0.2-0.5,0.4-0.9,0.5-1.4l0,0 c0.2-0.4,0.4-0.9,0.6-1.3l0,0c0.1-0.4,0.4-0.7,0.4-1.1c0.2-0.4,0.3-0.7,0.5-1.1c0.1-0.3,0.3-0.6,0.4-1c0.2-0.3,0.3-0.5,0.4-0.8 c0.1-0.2,0.2-0.4,0.3-0.7l0,0c0.1-0.3,0.2-0.6,0.4-0.8c0.1-0.3,0.2-0.6,0.4-0.9c0.1-0.2,0.2-0.5,0.3-0.7c0.1-0.2,0.2-0.5,0.3-0.7 c0.1-0.2,0.2-0.5,0.3-0.8c0.1-0.1,0.1-0.2,0.2-0.3c0.1-0.2,0.1-0.4-0.1-0.6c-0.3-0.3-0.5-0.4-0.9-0.2c-0.3,0.1-0.6,0.2-0.9,0.3 c-0.2,0-0.5,0.1-0.7,0.2l0,0l-0.7,0.2c-0.3,0.1-0.5,0.2-0.8,0.2l-0.8,0.2c-0.2,0.1-0.4,0.1-0.7,0.2v0c-0.3,0.1-0.6,0.2-0.8,0.3 l0,0l0,0l-0.8,0.2v0c-0.3,0.1-0.6,0.2-1,0.3c-0.3,0.1-0.6,0.2-0.9,0.3l0,0c-0.3,0.1-0.7,0.2-1,0.3c-0.2,0-0.4,0.1-0.6,0.2l0,0 c-0.3,0.1-0.5,0.2-0.8,0.2l0,0c-0.3,0.1-0.6,0.2-0.9,0.3c-0.3,0-0.5,0.1-0.8,0.2v0l0,0c-0.3,0.1-0.7,0.2-1,0.3v0 c-0.8,0.3-1.6,0.5-2.3,0.8c-0.4,0.1-0.8,0.2-1.2,0.4c-0.2,0.1-0.4,0.1-0.7,0.2l0,0c-0.3,0.1-0.6,0.2-0.9,0.3 c-0.2,0.1-0.5,0.1-0.7,0.2v0c-0.1,0-0.1,0-0.2,0.1v0c-0.2,0-0.4,0.1-0.5,0.2c-0.1,0-0.3,0.1-0.4,0.1c-0.1,0-0.3,0-0.4,0.1l0,0 c-0.1,0-0.1,0-0.2,0.1v0c-0.1,0-0.3,0.1-0.4,0.1v0c-0.1,0-0.1,0-0.2,0.1h0c-0.1,0-0.3,0-0.4,0.1l0,0c-0.3,0-0.5,0.1-0.8,0.2 c-0.3,0.1-0.5,0.1-0.8,0.2c-0.2,0-0.4,0.1-0.5,0.2c-0.3,0.1-0.5,0.1-0.8,0.2v0c-0.1,0-0.2,0-0.4,0.1v0c-0.2,0-0.4,0.1-0.6,0.2 c-0.2,0-0.4,0.1-0.6,0.2c-0.2,0-0.4,0.1-0.6,0.2c-0.3,0.1-0.6,0.2-0.9,0.3c-0.3,0-0.5,0.1-0.8,0.2v0c-0.2,0-0.3,0.1-0.5,0.1 c-0.1,0-0.2,0.1-0.2,0.2c0,0.3-0.2,0.7-0.3,1c-0.1,0.1-0.1,0.2,0,0.3c0.3,0.2,0.6,0.4,0.9,0.6v0c0.2,0.2,0.5,0.4,0.7,0.5v0 c0.2,0.2,0.5,0.4,0.8,0.6c0.2,0.2,0.5,0.4,0.7,0.5c0.2,0.2,0.4,0.3,0.7,0.5v0c0.1,0.1,0.3,0.3,0.5,0.4c0.1,0.1,0.3,0.3,0.5,0.4v0 c0.2,0.2,0.5,0.4,0.8,0.6c0.3,0.2,0.6,0.5,0.9,0.7v0l0.8,0.6v0v0c0.4,0.3,0.9,0.7,1.3,1c0.3,0.3,0.6,0.5,1,0.7 c0.2,0.2,0.5,0.4,0.7,0.5l0,0c0.7,0.6,1.4,1,2,1.6h0c0.2,0.2,0.4,0.3,0.7,0.5v0c0.2,0.2,0.3,0.3,0.5,0.4v0 c0.4,0.3,0.8,0.6,1.2,0.9c0.4,0.3,0.8,0.6,1.1,0.8c0.3,0.2,0.6,0.4,0.9,0.6c0.3,0.2,0.6,0.5,1,0.7c0.3,0.2,0.5,0.4,0.8,0.6 c0.1,0.1,0.2,0.1,0.3,0c0.3-0.2,0.7-0.3,1.1-0.3c0.7,0.1,0.9-0.3,1-0.8c0,0,0-0.1,0-0.1c0.2-0.3,0.3-0.7,0.4-1 c0.1-0.2,0.2-0.5,0.3-0.7l0,0c0.1-0.3,0.3-0.6,0.4-0.8c0.1-0.3,0.3-0.6,0.4-0.9c0.1-0.1,0.1-0.2,0.1-0.3c0.1-0.1,0.1-0.2,0.1-0.3 l0,0l0,0C1017.4,86.2,1017.4,86.2,1017.4,86L1017.4,86z" class="st8"/></g></g></svg>
```

Comment:
@Sudhish2101 where changes to this file intentional?

Reply:
@leecalcote Yes. I edited this so that the font is easily readable as mentioned in the issue.